src/main/java/frc/robot/commands/RunShooterToRPMCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.ShooterPrototypeSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class RunShooterToRPMCommand extends CommandBase {
+  
+  private final ShooterPrototypeSubsystem shooterSubsystem;
+
+  /**
+   * Creates a new RunShooterToRPMCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public RunShooterToRPMCommand(ShooterPrototypeSubsystem subsystem, double power) {
+    shooterSubsystem = subsystem;
+    shooterSubsystem.setTargetPower(power);
```

Comment:
This should be in initialize(). The constructor is called when the command is created with "new" at robot startup, initialize() is every time when the command is actually scheduled to run.